src/redux/books/books.js

```diff
@@ -1,39 +1,68 @@
-// Actions
-const ADD_BOOK = 'bookstore-app/books/ADD_BOOK';
-const REMOVE_BOOK = 'bookstore-app/books/REMOVE_BOOK';
-
-const initialState = [
-  {
-    id: 1,
-    title: '100 WAYS TO BE MONEY WISER',
-    author: 'Tilda Nabbanja',
-  },
-  {
-    id: 2,
-    title: 'BACK HOUSE',
-    author: 'Butamanya George',
-  },
-];
-
-export const addBook = (book) => ({
-  type: ADD_BOOK,
-  payload: book,
-});
-
-export const removeBook = (index) => ({
-  type: REMOVE_BOOK,
-  payload: index,
-});
-
-const reduceBook = (state = initialState, action) => {
+/* eslint-disable no-case-declarations */
+import * as api from '../../api/api';
+
+const ADD_BOOK = 'BookStores/books/ADD_BOOK';
+const REMOVE_BOOK = 'BookStores/books/REMOVE_BOOK';
+const GET_BOOKS = 'BookStores/books/GET_BOOKS';
+
+const handleData = (data) => {
+  const books = [];
+  const keys = Object.keys(data);
+
+  keys.forEach((key, index) => {
+    const book = data[keys[index]];
+    book[0].item_id = key;
+
+    books.push(book[0]);
+  });
+
+  return books;
+};
+
+// This shows how to implement Action Creators
+export const getBooks = () => async (dispatch) => {
```

Comment:
- [ ] This is a way to create an action creator, but the requirement suggests that you should use `createAsyncThunk`, kindly consider implementing this in all your action creators so your code respects the requirements